client/extensions/wp-super-cache/components/navigation/index.jsx

```diff
@@ -86,12 +84,20 @@ const Navigation = ( { activeTab, site, translate } ) => {
 
 Navigation.propTypes = {
 	activeTab: PropTypes.string,
-	site: PropTypes.object,
+	siteId: PropTypes.number,
+	// connected props
+	siteSlug: PropTypes.string,
 	translate: PropTypes.func.isRequired,
 };
 
 Navigation.defaultProps = {
 	activeTab: '',
 };
 
-export default localize( Navigation );
+const connectComponent = connect(
+	( state, { siteId } ) => ( {
+		siteSlug: getSiteSlug( state, siteId )
```

Comment:
Yeah, I was one the fence about this one. I was actually considering a third option, computing `siteSlug` in `Main`'s `connect`, and passing it down to `Navigation`.

I don't feel strongly about any of those. My motivation was, roughly, consistency with other child components of `Main` that also accept `siteId` as a prop -- the advantage being that (hopefully) not all of them will have to be `connect()`ed (tho I didn't check TBH).

> My only question is - do we actually need to pass down `siteId` from the controller at all ?

One minor clarification -- we aren't passing it down from the controller but only from `Main`. (We do compute a `siteId`-- actually a slug -- inside the controller, but only for use with analytics there.)